model/map/src/main/java/org/keycloak/models/map/user/MapUserProvider.java

```diff
@@ -49,14 +49,7 @@
 import org.keycloak.storage.UserStorageProvider;
 import org.keycloak.storage.client.ClientStorageProvider;
 
-import java.util.EnumMap;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
```

Comment:
please remove the star import. Its not allowed in Keycloak.

Reply:
@bs-matil Fixed. Thanks a lot !